### ISSUE TITLE
"Hide non-functional changes" should hide changes from the original file too

### ISSUE THREAD

**Issue by dimonchik0036** [JetBrains-Research/RefactorInsight#128]
Without the option:
<img width="1320" alt="image" src="https://user-images.githubusercontent.com/22503910/225047189-913e652c-f180-4972-b98c-202eb05b4c2f.png">
With the option:
<img width="1320" alt="image" src="https://user-images.githubusercontent.com/22503910/225047259-a9f51b23-645a-4925-afe8-f240c1f06646.png">

Another example:
<img width="1320" alt="image" src="https://user-images.githubusercontent.com/22503910/225047862-aa1ecad7-bad7-4147-91a6-5679f1602f34.png">
<img width="1320" alt="image" src="https://user-images.githubusercontent.com/22503910/225047941-855df0b9-61bb-43e2-9d80-65e92530d947.png">

These highlights on the left side (with the option) looks redundant

**Comment by onewhl** [comment]
@dimonchik0036 yes, it's a known bug, we didn't find a way to remove such highlighting from the left side of the diff.
We thought that setting `com.intellij.openapi.editor.FoldRegion#setInnerHighlightersMuted()` equal `true` might help, but it didn't. If you have any ideas on how to resolve it, please let us know :)
